tests/steps/test_steps.py

```diff
@@ -89,7 +89,15 @@
         "question": "Why is the sky blue?",
         "k": 5
     },
-    "output": [1, 3, 5, 7, 9],
+    "output": (
```

Comment:
I'm confused. Why is the output a tuple?

Reply:
It is a string formed by concatenating all the strings within the parentheses. This is a common way in Python to break up a long string into lines without messy `+` and `\`.